src/components/MovieDetails/index.tsx

```diff
@@ -111,6 +117,7 @@ const MovieDetails: React.FC<MovieDetailsProps> = ({ movie }) => {
     return <Error statusCode={404} />;
   }
 
+  const showAllStudios = data.productionCompanies.length <= minStudios + 1;
```

Comment:
Perhaps move this up to where the other new `const`s are?

Reply:
PR looks good to me though :)

Reply:
I don't think I can because `data` isn't guaranteed to be defined yet.

Reply:
Ah you're right, whoops! :P